metricbeat/module/vsphere/host/host.go

```diff
@@ -93,14 +97,19 @@ func (m *HostMetricSet) Fetch(ctx context.Context, reporter mb.ReporterV2) error
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	period := m.Module().Config().Period
+	if !isValidPeriod(period.Seconds()) {
+		return fmt.Errorf("invalid period %v. Please provide one of the following values: 20, 300, 1800, 7200, 86400", period)
+	}
```

Comment:
We should give user the metrics other than perf metrics, even if this validation fails.